crates/ibc-query/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name         = "ibc-query"
-version      = "0.45.0"
+version      = "0.46.0"
```

Comment:
Question. How are we distributing `ibc-query` package? If it is a separate package, do we still need to keep the same version number as `ibc`? (e.g. `ibc-derive` has different version number)

Reply:
Good question!

More of a common practice, and in a simplistic view `ibc-query` is dependent on `ibc` and this approach ensures that all dependencies are kept in sync. Also, for an easier map version for users.

As for the `ibc-derive` things are a bit different. That’s not directly dependent on `ibc` crate. we re-export everything in `ibc-derive`, so projects won’t need to explicitly have it as a dependency. Also, `ibc-derive` only needs to get updated if there are only changes to `ClientState` APIs.